reader/tests/system/filter/test_filter.py

```diff
@@ -27,7 +28,16 @@
 
 
 def setup_data(connection, cursor, deleted=False):
-    for fqid, model in list(data.items()) + list(other_models.items()):
+    all_models = list(data.items()) + list(other_models.items())
+    max_pos = max(m["meta_position"] for _, m in all_models)
+    cursor.execute(
+        "insert into positions (user_id) values " + ",".join(["(0)"] * max_pos)
```

Comment:
Same as in the other PR: Is it possible to not create a position entry for each position?